airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/MySqlSource.java

```diff
@@ -50,6 +49,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static io.airbyte.integrations.util.MySqlSslConnectionUtils.obtainConnection;
```

Comment:
this seems to be an unused import

Reply:
I double check, this is actually used, it seems that it was removed for some reason